packages/maskbook/src/plugins/GoodGhosting/UI/PersonalView.tsx

```
@@ -0,0 +1,154 @@
+import { formatBalance, formatEthereumAddress } from '@masknet/web3-shared'
+import { makeStyles, Grid, Typography, Button } from '@material-ui/core'
+import { useState } from 'react'
+import { useI18N } from '../../../utils'
+import { useEarlyWithdraw } from '../hooks/useGameActions'
+import { useGameToken } from '../hooks/usePoolData'
+import type { GoodGhostingInfo, Player } from '../types'
+import { getPlayerStatus, PlayerStatus } from '../utils'
+import BigNumber from 'bignumber.js'
+
+const useStyles = makeStyles((theme) => ({
+    infoRow: {
+        paddingBottom: theme.spacing(1),
+    },
+    circularDataSection: {
+        paddingTop: theme.spacing(2),
+    },
+    circularDataWrapper: {
+        minWidth: '80px',
+    },
+    circularData: {
+        padding: theme.spacing(1),
+        maxWidth: '100px',
+        margin: 'auto',
+    },
+    withdraw: {
+        marginTop: theme.spacing(5),
+        textAlign: 'center',
+    },
+}))
+
+interface PersonalViewProps {
+    info: GoodGhostingInfo
+}
+
+export function PersonalView(props: PersonalViewProps) {
+    const classes = useStyles()
+    const { t } = useI18N()
+    const gameToken = useGameToken()
+    const { canEarlyWithdraw, earlyWithdraw } = useEarlyWithdraw(props.info)
+    const [buttonEnabled, setButtonEnabled] = useState(true)
+
+    const status = useGetStatus(props.info, props.info.currentPlayer)
+
+    if (!props.info.currentPlayer) {
+        return (
+            <Typography variant="h6" color="textSecondary">
+                {t('plugin_good_ghosting_not_a_participant')}
+            </Typography>
+        )
+    }
+
+    const withdraw = async () => {
+        setButtonEnabled(false)
+        try {
+            await earlyWithdraw()
+        } catch (error) {
```

plz note why need ignore error

---

```suggestion
        } catch (error) {
            // ignore error
            // HACK: ....
```